joplin/templates/wagtailadmin/base.html

```diff
@@ -1,9 +1,19 @@
 {% extends "wagtailadmin/base.html" %}
-{% load staticfiles %}
+{% load staticfiles wagtailadmin_tags %}
 
 {% block branding_logo %}
-    <img src="{% static 'images/logo.png' %}" alt="City of Austin" width="80" />
-    <script>
-        window.JANIS_URL = '{{ JANIS_URL }}';
-    </script>
+{% endblock %}
+
+{% block js %}
+    {{ block.super }}
+
+    <joplinheader>
```

Comment:
Whats up with this element naming? Would it make more sense for this to be `<div class="joplinheader">`